pkg/minikube/cruntime/docker.go

```diff
@@ -89,16 +89,30 @@ func (r *Docker) ListContainers(filter string) ([]string, error) {
 	if err != nil {
 		return nil, err
 	}
+	content = strings.TrimSpace(content)
+	// Otherwise, strings.Split will return []string{""}
+	if content == "" {
+		return []string{}, nil
+	}
 	return strings.Split(content, "\n"), nil
 }
 
-// KillContainers forcibly removes a running pod based on ID
+// KillContainers forcibly removes a running container based on ID
 func (r *Docker) KillContainers(ids []string) error {
+	if len(ids) == 0 {
+		glog.Warningf("KillContainers was called with an empty list of ids, nothing to do.")
```

Comment:
do we need this warning? 
if yes, do we want to be consistent about it with `StopContainers`?

Reply:
and in crictl we don't have any warnings either.

Reply:
Thanks for catching this. I had meant to remove this.